.github/workflows/beam_PostCommit_Go_Dataflow_ARM.yml

```diff
@@ -55,14 +55,19 @@ jobs:
       github.event_name == 'schedule' ||
       github.event.comment.body == 'Run Go PostCommit Dataflow ARM'
     runs-on: [self-hosted, ubuntu-20.04, main]
-    name: beam_PostCommit_Go_Dataflow_ARM (Run Go PostCommit Dataflow ARM)
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_PostCommit_Go_Dataflow_ARM"]
+        job_phrase: ["Run Go PostCommit Dataflow ARM"]
     steps:
       - uses: actions/checkout@v3
       - name: Setup repository
         uses: ./.github/actions/setup-action
         with:
-          comment_phrase: 'Run Go PostCommit Dataflow ARM'
+          comment_phrase: ${{ matrix.job_phrase }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} ${{ matrix.job_phrase }}
```

Comment:
The brackets are missing here. Should be:
```
github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
```

Reply:
Fixed